indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamIndexTaskRunner.java

```diff
@@ -814,7 +814,13 @@ public void onFailure(Throwable t)
       catch (Exception e) {
         // (1) catch all exceptions while reading from kafka
         caughtExceptionInner = e;
-        log.error(e, "Encountered exception in run() before persisting.");
+        if (e instanceof InterruptedException || (e.getCause() != null
+                                                  && e.getCause() instanceof InterruptedException)) {
+          // InterruptedException stack trace is not that useful and will clog up the logs.
```

Comment:
```suggestion
        if (e instanceof InterruptedException || (Throwables.getRootCause(e) instanceof InterruptedException)) {
          // Suppress InterruptedException stack trace to avoid flooding the logs
```

Reply:
hi @kfaraz, Thanks for the suggestion!

Since InterruptedException does not have instantiator with a `Throwable`, the root cause of an InterruptedException is the InterruptedException itself. I have tidied the line to only be `Throwables.getRootCause(e) instanceof InterruptedException`.